packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -137,21 +137,23 @@ export function ArticleMetadataPanel({value, infoData, onClose, onChange}: Artic
               </FormGroup>
               <FormGroup>
                 <ControlLabel>{t('articleEditor.panels.socialMediaTitle')}
-                <label style={{float: 'right'}}> {value.socialMediaTitle?.length}</label>
+                {socialMediaTitle ? <label style={{float: 'right'}}> {value.socialMediaTitle?.length}/{socialMediaTitleMax}</label> :
```

Comment:
Another way to write this would be:

`<label style={{float: 'right'}}> {value.socialMediaTitle ? value.socialMediaTitle.length : 0}/{socialMediaTitleMax}</label>`

Makes the whole thing a bit more readable.